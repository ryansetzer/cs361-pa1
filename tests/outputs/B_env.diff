--- outputs/B_env.txt	2022-09-25 14:55:18.854678929 -0400
+++ expected/B_env-2.txt	2022-09-11 14:59:21.000000000 -0400
@@ -0,0 +1,10 @@
+A=5
+A=5
+B=6
+C=7
+A=5
+$ ./bin/env A=5 ./bin/env
+$ ./bin/env A=5 B=6 C=7 ./bin/env
+$ ./bin/env A=5 B=6 C=7 ./bin/env | head -n 1
+$ quit
+
