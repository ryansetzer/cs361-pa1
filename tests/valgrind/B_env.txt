==59403== Memcheck, a memory error detector
==59403== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==59403== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==59403== Command: ../dukesh -b scripts/env.txt
==59403== 
==59403== 
==59403== HEAP SUMMARY:
==59403==     in use at exit: 0 bytes in 0 blocks
==59403==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==59403== 
==59403== All heap blocks were freed -- no leaks are possible
==59403== 
==59403== For lists of detected and suppressed errors, rerun with: -s
==59403== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
