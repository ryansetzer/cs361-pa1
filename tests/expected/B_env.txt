$ ./bin/env A=5 ./bin/env
A=5
$ ./bin/env A=5 B=6 C=7 ./bin/env
A=5
B=6
C=7
$ ./bin/env A=5 B=6 C=7 ./bin/env | head -n 1
A=5
$ quit

